### ISSUE TITLE
v1.1.2; local track added to database again when played

### ISSUE THREAD

**Issue by erkr** [music-assistant/hass-music-assistant#166]
I imported a collection of local files. Then I selected the first track to play and noticed my number of tracks increased by 1, and the selected track is in the list twice:
![image](https://user-images.githubusercontent.com/4363599/168495307-0b98b8ad-cc64-4265-9413-cf0120e2277f.png)

this error was related:
```
Logger: music_assistant.mass
Source: /usr/local/lib/python3.9/site-packages/music_assistant/mass.py:243
First occurred: 23:43:39 (16 occurrences)
Last logged: 23:51:38

Job [Add file://album/zakkwyldebookofshadows to database] failed with error 'NoneType' object has no attribute 'name'.
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/music_assistant/controllers/music/albums.py", line 81, in add
    await self._match(db_item)
  File "/usr/local/lib/python3.9/site-packages/music_assistant/controllers/music/albums.py", line 236, in _match
    if await find_prov_match(provider):
  File "/usr/local/lib/python3.9/site-packages/music_assistant/controllers/music/albums.py", line 205, in find_prov_match
    searchstr = f"{db_album.artist.name} {db_album.name}"
AttributeError: 'NoneType' object has no attribute 'name'
```

**Comment by marcelveldt** [comment]
The seconds error should resolved in the 1.1.3 release.
The duplicate tracks bug I'm currently investigating.

**Comment by marcelveldt** [comment]
fixed in 1.1.4 - library rescan will be auto performed

**Comment by erkr** [comment]
Yep, thanks 😊
